### PR TITLE
Fix/general fixes

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: app/Base/components/Navbar/index.tsx [hunk 1]
```diff
@@ -79,30 +79,29 @@ function Navbar(props: Props) {
 
     const profileUrl = useRouteMatching(routes.myProfile);
 
-    const [logout] = useMutation<LogoutMutation, LogoutMutationVariables>(
+    const [
+        logout,
+        // { loading: logoutLoading },
```
Comment (AdityaKhatri) [inline app/Base/components/Navbar/index.tsx:84]: why add this just to not use?

File: app/Base/types/project.ts [hunk 2]
```diff
@@ -1,5 +1 @@
-export type Project = {
```
Comment (AdityaKhatri) [inline app/Base/types/project.ts:1]: This is not required right? Lets remove this.

File: app/views/BookDetail/index.tsx [hunk 3]
```diff
@@ -156,25 +155,29 @@ mutation DeleteCartItem($id: ID!) {
 `;
 
 function BookDetail() {
-    const { id } = useParams();
+    // const { id } = useParams<{ id: string }>();
```
Comment (AdityaKhatri) [inline app/views/BookDetail/index.tsx:158]: remove this line

File: app/views/HomePage/index.tsx [hunk 4]
```diff
@@ -175,14 +191,24 @@ function HomePage() {
...
                         className={styles.featuredBooksSection}
                         heading={strings.featuredBooksLabel}
+                        footerContent={(
+                            <Pager
+                                activePage={page}
+                                maxItemsPerPage={pageSize}
+                                itemsCount={result?.books?.totalCount ?? 0}
+                                onActivePageChange={setPage}
+                                onItemsPerPageChange={setPageSize}
+                                itemsPerPageControlHidden
```
Comment (AdityaKhatri) [inline app/views/HomePage/index.tsx:199]: why hide this if onchange is handled?

File: app/views/Login/LoginForm/index.tsx [hunk 5]
```diff
@@ -119,37 +115,37 @@ function LoginForm() {
...
-    const handleSubmit = useCallback((finalValue) => {
-        // elementRef.current?.resetCaptcha();
+    const handleSubmit = useCallback((finalValue: PartialFormType) => {
+        const formValue = finalValue as FormType;
         login({
-            variables: {
-                email: finalValue?.email,
-                password: finalValue?.password,
-            },
+            variables: formValue,
```
Comment (AdityaKhatri) [inline app/views/Login/LoginForm/index.tsx]: why not directly cast:
variables: formValue as FormType,

File: package.json [hunk 6]
```diff
@@ -23,15 +24,15 @@
...
     },
     "keywords": [],
     "author": "",
     "license": "ISC",
     "bugs": {
-        "url": "https://github.com/togglecorp/base-react-app/issues"
+        "url": "https://github.com/togglecorp/kitab-client/issues"
     },
-    "homepage": "https://github.com/togglecorp/base-react-app#readme",
+    "homepage": "https://github.com/togglecorp/kitab-client#readme",
```
Comment (AdityaKhatri) [inline package.json]: all of these have been changed

File: app/views/WishList/index.tsx [hunk 7]
```diff
@@ -314,29 +320,44 @@ function WishList(props: Props) {
...
                     )}
                     filtered={false}
                 />
+                <Pager
+                    activePage={page}
+                    maxItemsPerPage={pageSize}
+                    itemsCount={data?.wishList?.totalCount ?? 0}
+                    onActivePageChange={setPage}
+                    onItemsPerPageChange={setPageSize}
+                    itemsPerPageControlHidden
```
Comment (AdityaKhatri) [inline app/views/WishList/index.tsx]: doesn't make sense

File: app/views/Profile/IndividualProfile/index.tsx [hunk 8]
```diff
@@ -235,6 +239,16 @@ function IndividualProfile(props: Props) {
...
                         </SmartButtonLikeLink>
                     )}
+                    footerContent={(
+                        <Pager
+                            activePage={page}
+                            maxItemsPerPage={pageSize}
+                            itemsCount={orderList?.orders?.totalCount ?? 0}
+                            onActivePageChange={setPage}
+                            onItemsPerPageChange={setPageSize}
+                            itemsPerPageControlHidden
```
Comment (AdityaKhatri) [inline app/views/Profile/IndividualProfile/index.tsx]: doesn't make sense

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: app/Base/components/Navbar/index.tsx (hunk 1 above)
Comment (tnagorra) [reply app/Base/components/Navbar/index.tsx:84]: We will use this later. One of the component cannot be disabled

File: app/Base/types/project.ts (hunk 2 above)
Comment (tnagorra) [reply app/Base/types/project.ts:1]: This is used in one of the check permissions function

File: app/views/BookDetail/index.tsx (hunk 3 above)
Comment (tnagorra) [reply app/views/BookDetail/index.tsx:158]: Let's not remove that line. It is how the function should be used.
It's kept as a reference for the time when typings will be fixed

File: package.json (hunk 6 above)
Comment (tnagorra) [reply package.json]: oh no!

File: app/views/Profile/IndividualProfile/index.tsx (hunk 8 above)
Comment (tnagorra) [reply app/views/Profile/IndividualProfile/index.tsx]: what? setting page size?

File: app/views/WishList/index.tsx (hunk 7 above)
Comment (tnagorra) [reply app/views/WishList/index.tsx]: what? setting page size?

File: app/views/Profile/IndividualProfile/index.tsx (hunk 8 above)
Comment (AdityaKhatri) [reply app/views/Profile/IndividualProfile/index.tsx]: yes

**Pull request merged**